.github/workflows/build-publish.yml

```diff
@@ -0,0 +1,184 @@
+name: Build and publish Node binaries
+
+on:
+  workflow_dispatch:
+
+jobs:
+  linux-x64:
+    runs-on: ubuntu-20.04
+
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Setup node
+        uses: actions/setup-node@v1
+        with:
+          node-version: '14.x'
+
+      - name: Build nodejs
+        run: |
+          npm install
+          npm run start
+
+      - uses: actions/upload-artifact@v2
+        with:
+          name: linux-x64
+          path: dist
+
+  linux-arm64:
+    runs-on: ubuntu-20.04
+
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v1
+
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v1
+
+      - name: Build
+        uses: docker/build-push-action@v2
+        with:
+          context: .
+          file: ./docker/Dockerfile.linux
+          platforms: linux/arm64
+          outputs: type=tar,dest=../out.tar
+
+      - name: Extract binaries from Docker image
+        run: |
+          tar xvf ../out.tar root/pkg-fetch/dist
+          mv root/pkg-fetch/dist dist
+
+      - uses: actions/upload-artifact@v2
+        with:
+          name: linux-arm64
+          path: dist
+
+  linuxstatic-x64:
+    runs-on: ubuntu-20.04
+
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v1
+
+      - name: Build
+        uses: docker/build-push-action@v2
+        with:
+          context: .
+          build-args: |
+            TARGET_TOOLCHAIN_ARCH=x86_64
+            TARGET_NODE_ARCH=x64
+          file: ./docker/Dockerfile.linuxstatic
+          platforms: linux/amd64
+          outputs: type=tar,dest=../out.tar
+
+      - name: Extract binaries from Docker image
+        run: |
+          tar xvf ../out.tar root/pkg-fetch/dist
+          mv root/pkg-fetch/dist dist
+      - uses: actions/upload-artifact@v2
+        with:
+          name: linuxstatic-x64
+          path: dist
+
+  linuxstatic-arm64:
+    runs-on: ubuntu-20.04
+
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v1
+
+      - name: Build
+        uses: docker/build-push-action@v2
+        with:
+          context: .
+          build-args: |
+            TARGET_TOOLCHAIN_ARCH=aarch64
+            TARGET_NODE_ARCH=arm64
+          file: ./docker/Dockerfile.linuxstatic
+          platforms: linux/arm64
```

Comment:
This is a cross toolchain. Don’t emulate arm64. Use x86-64 for toolchain.

Reply:
I have no experience in this, could you fix it?